tuist/tuist
# Skip implicit import false positive when import is commented out

Resolves part of https://github.com/tuist/tuist/issues/6706

### Short description 📝

Added support for comments in code. Now one line and multiline comments are supported in SourceScanner.

### How to test the changes locally 🧐

remove dependency from tuist manifest -> comment some import -> launch inspect implicit-imports -> and all will be ok

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/get-started) for reference).

### Contributor checklist ✅


### Reviewer checklist ✅
